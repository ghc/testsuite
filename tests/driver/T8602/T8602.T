test('T8602', extra_clean(['t8602.sh']), ghci_script, ['T8602.script'])
