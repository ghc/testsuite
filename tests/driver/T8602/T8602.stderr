A B C
phase `Haskell pre-processor' failed (exitcode = 1)
